modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.js

```diff
@@ -38,82 +90,120 @@ const SetupAnalyticsCloudPage = ({handlePage, leftButton}) => {
-					label="Time Zone"
-					name="analytics.timeZone"
-					placeholder="UTC-04:00"
-					type="text"
-				/>
-
-				<Input
-					groupStyle="pb-1"
-					helper="This user will be the recepient of any high priority communications."
-					label="Incident Report Contact"
-					name="analytics.incidentReportContact"
-					placeholder="user@company.com"
-					required
-					type="email"
-				/>
-			</ClayForm.Group>
-
-			<Button
-				className="btn-outline-primary ml-3 my-2 rounded-xs"
-				prependIcon="plus"
-				small
-			>
-				Add Incident Report Contact
-			</Button>
+			<FieldArray
+				name="activations.incidentReportContact"
+				render={({pop, push}) => (
+					<>
+						<ClayForm.Group className="mb-0">
+							<ClayForm.Group className="mb-0">
+								<Input
+									groupStyle="pb-1"
+									helper="This user will create and manage the Analytics Cloud Workspace and must have a liferay.com account. The owner Email can be updated vis Support ticket if needed."
+									label="Owner Email"
+									name="activations.ownerEmail"
+									placeholder="user@company.com"
+									required
+									type="email"
+								/>
+
+								<Input
+									groupStyle="pb-1"
+									helper="Lowercase letters and numbers only. Project IDs cannot be changed."
+									label="Workspace Name"
+									name="activations.workspaceName"
+									placeholder="superbank1"
+									required
+									type="text"
+									validations={[
+										(value) => maxLength(value, 255),
+										(value) => isLowercaseAndNumbers(value),
+									]}
+								/>
+
+								<Select
+									groupStyle="pb-1"
+									helper="Select a server location for your data to be stored."
+									key={analyticsDataCenterRegions}
+									label="Data Center Location"
+									name="activations.dataCenterLocation"
+									options={analyticsDataCenterRegions}
+									required
+								/>
+
+								<Input
+									groupStyle="pb-1"
+									helper="Please note that the friendly URL cannot be changed once added."
+									label="Workspace Friendly URL"
+									name="activations.workspaceURL"
+									placeholder="/myurl"
+									type="text"
+									validations={[
+										(value) => hasFriendlyURL(value),
+										(value) => hasNoEmptySpace(value),
+										(value) => isStartingWithSlash(value),
+									]}
+								/>
+
+								<Input
+									groupStyle="pb-1"
+									helper="Anyone with an email address at the provided domains can request access to your Workspace. If multiple, separate domains by commas."
+									label="Allowed Email Domains"
+									name="activations.allowedEmailDomains"
+									placeholder="@mycompany.com"
+									type="email"
+								/>
+
+								<Input
+									groupStyle="pb-1"
+									helper="Enter the timezone to be used for all data reporting in your Workspace."
+									label="Time Zone"
+									name="activations.timeZone"
+									placeholder="UTC-04:00"
+									type="text"
+								/>
```

Comment:
Hey @claricebrasil. Reduce the code here creating a component for those Inputs and Selects and call them here. eg: <AnalyticsCloudInputs ...>